pilot/pkg/xds/debug.go

```diff
@@ -97,7 +97,7 @@ type AdsClient struct {
 	ConnectionID string              `json:"connectionId"`
 	ConnectedAt  time.Time           `json:"connectedAt"`
 	PeerAddress  string              `json:"address"`
-	Metadata     *model.NodeMetadata `json:"metadata"`
+	Metadata     *model.NodeMetadata `json:"metadata,omitempty"`
```

Comment:
When can this field be empty

Reply:
when /connections is displayed https://github.com/istio/istio/blob/master/pilot/pkg/xds/debug.go#L511